cmd/gaia/app/genesis.go

```diff
@@ -81,30 +85,49 @@ type GaiaGenTx struct {
 	PubKey  string         `json:"pub_key"`
 }
 
-// Generate a gaia genesis transaction with flags
-func GaiaAppGenTx(cdc *wire.Codec, pk crypto.PubKey, genTxConfig config.GenTx) (
-	appGenTx, cliPrint json.RawMessage, validator tmtypes.GenesisValidator, err error) {
+// GaiaAppGenTx generates a Gaia genesis transaction.
+func GaiaAppGenTx(
+	cdc *wire.Codec, pk crypto.PubKey, genTxConfig config.GenTx,
+) (json.RawMessage, json.RawMessage, tmtypes.GenesisValidator, error) {
```

Comment:
kinda minor - but I'm not really into this expanded format - the top like doesn't seem to be that big?

Reply:
Also I think removing the variable names from the return fields is a bad idea! - Those names provide context to the caller of the function so the developer doesn't need to look into the function to understand what is being called.

Reply:
@rigelrozanski are you saying the function sig of `GaiaAppGenTx` is not that long? I think it was over 120 columns iirc -- that's pretty long

re, removing named return vars. I have to respectfully disagree. I'm not sure how they add context? For me personally, I find large and/or complex functions extremely hard and confusing to read and understand with named params. I have to keep in mind everywhere a variable(s) was modified or completely changed and with empty returns all over the place, idk what is what.

Golang itself recommends this:
```
Naked return statements should be used only in short functions, as with the example shown here. They can harm readability in longer functions.
```

Reply:
I agree with golang re naked returns, but I do prefer named return values. Makes what you expect out of the function super clear at a glance.

Reply:
Sorry @rigelrozanski, I miss understood you initially. Yes, I think we're seeing eye-to-eye here. Named return variables do indeed help understanding the scope of the function/method...I just strongly dislike naked returns.

I'll revert the header part back 👍